src/components/ProfileInfo/ProfileInfo.tsx

```diff
@@ -0,0 +1,78 @@
+import Button from 'components/core/Button/Button';
+import ProfileInfoDialog from 'components/ProfileInfoDialog/ProfileInfoDialog';
+import UserModel from 'models/UserModel';
+import React, { useEffect, useState } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+import tokenService from 'services/tokenService';
+import userService from 'services/userService';
+import classes from './ProfileInfo.module.scss';
+
+const ProfileInfo = () => {
+  const [user, setUser] = useState<UserModel>({} as UserModel);
+  const location = useLocation();
+  const id: number = parseInt(location.pathname.split('/')[2]);
```

Comment:
There is no need to parse path to get id there is hook in react-router that can give this value to you